specification/network/resource-manager/Microsoft.Network/stable/2019-07-01/virtualRouter.json

```diff
@@ -0,0 +1,699 @@
+        "tags": [
+          "VirtualRouters"
+        ],
+        "operationId": "VirtualRouters_UpdateTags",
+        "description": "Updates a Virtual Router.",
+        "parameters": [
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The resource group name of the Virtual Router."
+          },
+          {
+            "name": "virtualRouterName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the Virtual Router being updated."
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "./network.json#/definitions/TagsObject"
+            },
+            "description": "Parameters supplied to Update Virtual Router Tags."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Request successful. Returns the details of the Virtual Router updated.",
+            "schema": {
+              "$ref": "#/definitions/VirtualRouter"
+            }
+          },
+          "default": {
+            "description": "Error.",
+            "schema": {
+              "$ref": "./network.json#/definitions/Error"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Update VirtualRouter": {
+            "$ref": "./examples/VirtualRouterUpdateTags.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "VirtualRouters"
+        ],
+        "operationId": "VirtualRouters_CreateOrUpdate",
+        "description": "Creates or updates the specified Virtual Router.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "virtualRouterName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the Virtual Router."
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/VirtualRouter"
+            },
+            "description": "Parameters supplied to the create or update Virtual Router."
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
```

Comment:
>responses [](start = 9, length = 9)

default response missing

Reply:
same in other apis

---
In reply to: [317344690](https://github.com/Azure/azure-rest-api-specs/pull/7005#discussion_r317344690) [](ancestors = 317344690)